index.html

```diff
@@ -55,29 +55,73 @@
     <!-- Main jumbotron for a primary marketing message or call to action -->
     <div class="jumbotron">
       <div class="container">
-        <h1>Hello, world!</h1>
-        <p>This is a template for a simple marketing or informational website. It includes a large callout called a jumbotron and three supporting pieces of content. Use it as a starting point to create something more unique.</p>
-        <p><a class="btn btn-primary btn-lg" href="#" role="button">Learn more &raquo;</a></p>
+        <div class="row">
+          <h1 class="text-center">Popular Categories</h1>
+        </div>
+        <div class="row">
+          <div class="col-md-3">
+            <a href="#" class="thumbnail">
+              <img src="images/placeholder.png" alt="placeholder image">
+            </a>
+          </div>
+          <div class="col-md-3">
+            <a href="#" class="thumbnail">
+              <img src="images/placeholder.png" alt="placeholder image">
+            </a>
+          </div>
+          <div class="col-md-3">
+            <a href="#" class="thumbnail">
+              <img src="images/placeholder.png" alt="placeholder image">
+            </a>
+          </div>
+          <div class="col-md-3">
+            <a href="#" class="thumbnail">
+              <img src="images/placeholder.png" alt="placeholder image">
+            </a>
+          </div>
+        </div>
       </div>
     </div>
 
     <div class="container">
       <!-- Example row of columns -->
       <div class="row">
-        <div class="col-md-4">
-          <h2>Heading</h2>
-          <p>Donec id elit non mi porta gravida at eget metus. Fusce dapibus, tellus ac cursus commodo, tortor mauris condimentum nibh, ut fermentum massa justo sit amet risus. Etiam porta sem malesuada magna mollis euismod. Donec sed odio dui. </p>
-          <p><a class="btn btn-default" href="#" role="button">View details &raquo;</a></p>
+        <h1 class="text-center">New &amp; Popular</h1>
+        <div class="media">
+          <div class="media-left">
+            <a href="#">
+              <img class="media-object" src="images/placeholder-200.png" alt="placeholder image">
+            </a>
+          </div>
+          <div class="media-body">
+            <h4 class="media-heading">Happy Valley</h4>
+            <p class="poem-info"><span class="author">Alice Lyons</span><span class="category">Nature</span><span class="rating">&#9734;&#9734;&#9734;&#9734;</span><span class="add-favorite pull-right glyphicon glyphicon-plus"></span></p>
```

Comment:
We need additional spacing between the "author," "category," and "rating" classes.